src/functionalTests/resources/json/bulkscan/formA.json

```diff
@@ -201,12 +201,6 @@
     { "name": "ApplicantRepresented", "value": "I am not represented by a solicitor in these proceedings" },
     { "name": "ApplicantSolicitorName", "value": "Saul Call" },
     { "name": "ApplicantSolicitorFirm", "value": "Better Divorce Ltd" },
-    { "name": "ApplicantSolicitorAddressLine1", "value": "20 Solicitors Road" },
```

Comment:
Shouldn't you have added the new fields to this file? i.e. Applicant's and respondent's line 1 and postcode.

Reply:
And if you didn't, shouldn't your tests be failing?

Reply:
The fields already exist "ApplicantAddressLine1", "ApplicantAddressPostcode", "RespondentAddressLine1" & "RespondentAddressPostcode" in this test data file. The ones I removed were just redundant test data that was never used anywhere (unsure why they were added in the first place)

Reply:
Got you. The ones you're removing are regarding the solicitors, not the citizens.
If that's what you want, I'm happy with it.